### DISCUSSION TITLE
ScriptReference/ScreenCapture.CaptureScreenshot

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#138] [Notes]
# ScriptReference/ScreenCapture.CaptureScreenshot

https://docs.unity3d.com/ScriptReference/ScreenCapture.CaptureScreenshot.html

**Comment by vertxxyz** [comment]
If you provide a full path you should include the `.png` extension in the `filename` parameter.

**Comment by qwe321qwe321qwe321** [comment]
In some cases, this method does not work properly with Super Size. You will still get the blurred detail.
Use Camera.targetTexture with super-size render texture instead.
